src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.h

```diff
@@ -0,0 +1,209 @@
+/**
+ * The Visualizer messenger is a singleton object that receives draw calls
+ * to draw shapes such as ellipse and rectangle.
+ *
+ * The messenger constructs and queues the shapes into shape messages and layer
+ * messages to be sent via ROS visualizer topics (VISUALIZER_DRAW_LAYER_TOPIC)
+ *
+ * Lasted edited by Muchen He on 2019-01-28
+ */
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <string>
+
+#include "thunderbots_msgs/DrawLayer.h"
+#include "thunderbots_msgs/DrawShape.h"
```

Comment:
`<ros/ros.h>` isn't being included here and is causing CI build failures. Remember to try build locally before pushing.

Reply:
Good catch.

O.o I did `catkin_make` before every commit. Maybe I had a dirty build?

Reply:
Funky, but no worries, that's why we have CI!